cmd/vcluster/cmd/start.go

```diff
@@ -469,18 +467,6 @@ func syncKubernetesService(ctx *context2.ControllerContext) error {
 
 		return errors.Wrap(err, "sync kubernetes service")
 	}
-
-	err = endpoints.SyncKubernetesServiceEndpoints(ctx.Context, ctx.VirtualManager.GetClient(), ctx.CurrentNamespaceClient, ctx.CurrentNamespace, ctx.Options.ServiceName)
```

Comment:
Can we safely remove this? Why is this not needed anymore now?

Reply:
I think yes. Because the same thing is now taken care by the new controller.